ports/servo/browser.rs

```diff
@@ -259,8 +259,12 @@ impl Browser {
                 EmbedderMsg::ResizeTo(_browser_id, size) => {
                     self.window.set_inner_size(size);
                 }
-                EmbedderMsg::Alert(_browser_id, message, sender) => {
-                    display_alert_dialog(message.to_owned(), sender);
+                EmbedderMsg::Alert(browser_id, message, sender) => {
+                    display_alert_dialog(message.to_owned());
```

Comment:
I'm hoping "joining" the thread will make this block until the use has dismissed the alert...